src/myst/interpreter/native_lib/time.cr

```diff
@@ -0,0 +1,49 @@
+module Myst
+  class Interpreter
+    NativeLib.method :time_now, Value do
```

Comment:
Maybe a nitpick, and I don't have a better idea, but since this is a static method, is there some way to show that in the name instead of just `:time_now`?

This is the first time a type is gaining a static method, so there's no precedent in place to follow :/